src/Component/src/Metadata/ResourceMetadata.php

```diff
@@ -36,13 +39,23 @@ public function __construct(
         ?array $operations = null,
     ) {
         $this->operations = null === $operations ? null : new Operations($operations);
+
+        if (null !== $driver && false !== $driver) {
+            trigger_deprecation('sylius/resource', '1.13', 'Using driver is deprecated. If your resource is managed by Doctrine you have nothing to do, otherwise use a customer provider.');
```

Comment:
Message needs to be re-worked, for now just marking error in `customer/custom` part.

```suggestion
            trigger_deprecation(
                'sylius/resource',
                '1.13',
                'Using driver is deprecated. If your resource is managed by Doctrine you have nothing to do, otherwise use a custom provider.'
            );
```